fusor-ember-cli/app/mixins/needs-deployment-mixin.js

```diff
@@ -10,5 +10,18 @@ export default Ember.Mixin.create({
 
   isNew: false,
 
-  deploymentName: Ember.computed.alias("deploymentController.model.name")
+  deploymentName: Ember.computed.alias("deploymentController.model.name"),
+
+  ////////////////////////////////////////////////////////////
+  // ALIASES AND COMMONLY USED COMPUTED PROPS
+  // Consolidates these and makes them available for free to any mixee
+  // Prevents littering leaf controllers with duplicated aliases
+  ////////////////////////////////////////////////////////////
+  upstreamConsumerUuid: Ember.computed.alias(
+    'deploymentController.model.upstream_consumer_uuid'),
```

Comment:
We have a lot of duplicate aliases throughout the project that are the same, this turned out to be a nice way to clean that up. Curious for feedback @cfchase @isratrade